laws/src/main/scala/cats/laws/discipline/ApplicativeTests.scala

```diff
@@ -54,9 +57,15 @@ trait ApplicativeTests[F[_]] extends ApplyTests[F] {
       "applicative map" -> forAll(laws.applicativeMap[A, B] _),
       "applicative unit" -> forAll(laws.applicativeUnit[A] _),
       "ap consistent with product + map" -> forAll(laws.apProductConsistent[A, B] _),
+      "replicateA_ consistent with replicateA.void" -> forAll { (a: A) =>
+        // Should be an implicit parameter but that is not a binary-compatible change
```

Comment:
Is there some place that these bincompat hacks are being catalogued? Should we make this a `// TODO: cats 3.x` comment?